service-map/src/main/java/fi/nls/oskari/map/layer/formatters/LayerJSONFormatterWFS.java

```diff
@@ -62,13 +63,9 @@ public JSONObject getJSON(OskariLayer layer,
     /**
      * Constructs a style json
      *
-     * @param  options wfs layer configuration
      */
-    private JSONArray getStyles(JSONObject options) {
-
-        JSONArray arr = new JSONArray();
-        // TODO: parse styles from options
-        return arr;
+    protected void setStyles(JSONObject layerJson, WFSLayerOptions wfsOpts) {
+        JSONHelper.putValue(layerJson, KEY_STYLES, wfsOpts.getStyles().names());
```

Comment:
`WFSLayerOptions#getStyles()` seems to edit the actual JSONObject of OskariLayer if the options doesn't have `styles` but I'm not sure if that's a big deal.